src/utils/mouse-events-handler.js

```diff
@@ -1,10 +1,14 @@
 // TODO: Make look speed adjustable by the user
 const HORIZONTAL_LOOK_SPEED = 0.1;
 const VERTICAL_LOOK_SPEED = 0.06;
+const VERTICAL_SCROLL_TIMEOUT = 250;
+const HORIZONTAL_SCROLL_TIMEOUT = 250;
 
 export default class MouseEventsHandler {
   constructor(cursor, cameraController) {
     this.cursor = cursor;
+    const cursorController = this.cursor.el.getAttribute("cursor-controller");
+    this.superHand = cursorController.cursor.components["super-hands"];
```

Comment:
Ohh this line confused me until I realized you were referring to the `cursor`'s `cursor-controller`'s `cursor`'s `super-hands` component, where the `cursor` started as the `input-configurator`'s `cursorController` selector-ed element's `cursor-controller` component -- silly me.